lib/stripe.js

```diff
@@ -24,6 +24,9 @@ Stripe.USER_AGENT_SERIALIZED = null;
 var APP_INFO_PROPERTIES = ['name', 'version', 'url'];
 
 var exec = require('child_process').exec;
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+var objectAssign = require('object-assign');
```

Comment:
Honestly, it's because I like to sort them in ascending (or sometimes descending) order of line length because I find it pleasing to look at.  ¯\\\_(ツ)_/¯

Will switch to the bottom one. :)